src/sentry/testutils/cases.py

```diff
@@ -178,6 +179,7 @@ def make_request(self, user=None, auth=None, method=None):
 
     # TODO(dcramer): ideally superuser_sso would be False by default, but that would require
     # a lot of tests changing
+    @TimedRetryPolicy.wrap(timeout=5)
```

Comment:
Does this actually need to be 5 seconds? Would 1 work ok?

Reply:
5 was an arbitrary choice. We can try 1 second and go up from there.

Reply:
Let's keep it at 5 for now, this shouldn't have too much of an affect on build times as this should only run when `login_as` fails. It's also been hard to reproduce/test due to CI flakyness, but I think we can check to see that these issues drop to near 0 https://sentry.io/organizations/sentry/issues/?project=2423079&query=is%3Aunresolved+HTTPConnectionPool&statsPeriod=14d